app/views/home/index.html.erb

```diff
@@ -1,14 +1,19 @@
 <h1>Welcome to Relocate</h1>
-<p>Moving to a new place is as exciting as it is stressful. Relocate was developed to help alleviate the stress so you can focus on settling in. What makes a home isn’t just the roof you sleep under, but the neighborhood around you. The average person moves approximately 11.7 times in their lifetime, and that means new gyms, new utilities, and new health care providers, among other things. Relocate's user-friendly interface is a quick way to find and compare all the local businesses and necessities around you based on the location you provide.</p>
+
 <div class="topnav">
-  <a><%= link_to 'The Team' %></a>
+  <a><%= link_to 'The Team', '/team' %></a>
   <a><%= link_to 'Contact' %></a>
   <a><%= link_to 'Help' %></a>
   <%= link_to 'Privacy Policy', "/privacy" %>
 </div>
 
 <div class="body">
-
+<br>
+<p>Moving to a new place is as exciting as it is stressful. Relocate was developed to help alleviate the stress so you
```

Comment:
This is great! I was reading back through it and thought maybe a minor revision for the first sentence could be "Moving to a new place can be exciting, but also stressful."

Reply:
Great, I'll change that. Thanks!